opendistro-elasticsearch-sql.release-notes

```diff
@@ -1,4 +1,17 @@
-## 2019-06-21, Version 1.1.0 (Current)
+## 2019-07-23, Version 1.2.0
```

Comment:
Should we add `(Current)` here as well ?

Reply:
Good catch, done